From: =?utf-8?q?Jonas_=C3=85dahl?= <jadahl@gmail.com>
Date: Wed, 13 Apr 2022 12:03:27 +0200
Subject: startup-notification: Fix some introspection annotations

This avoids a bunch of warnings and fixes a transfer annotation.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2374>
Origin: upstream, 42.1, commit:8c656a7f40515f3aef8303743a770396e25a01eb
---
 src/core/startup-notification.c      |  2 +-
 src/meta/meta-startup-notification.h | 18 +++++++++---------
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/core/startup-notification.c b/src/core/startup-notification.c
index 1a3dab5..bf46f43 100644
--- a/src/core/startup-notification.c
+++ b/src/core/startup-notification.c
@@ -701,7 +701,7 @@ meta_startup_notification_new (MetaDisplay *display)
  *
  * Get the list of startup sequences arrived in the startup notification object.
  *
- * Returns: (element-type MetaStartupSequence): a #GSList of
+ * Returns: (element-type MetaStartupSequence) (transfer none): a #GSList of
  * #MetaStartupSequence in the #MetaStartupNotification.
  **/
 GSList *
diff --git a/src/meta/meta-startup-notification.h b/src/meta/meta-startup-notification.h
index 85ef1a3..0bda6fe 100644
--- a/src/meta/meta-startup-notification.h
+++ b/src/meta/meta-startup-notification.h
@@ -44,30 +44,30 @@ MetaLaunchContext *
              meta_startup_notification_create_launcher (MetaStartupNotification *sn);
 
 META_EXPORT
-const char * meta_startup_sequence_get_id             (MetaStartupSequence *sequence);
+const char * meta_startup_sequence_get_id             (MetaStartupSequence *seq);
 
 META_EXPORT
-gboolean     meta_startup_sequence_get_completed      (MetaStartupSequence *sequence);
+gboolean     meta_startup_sequence_get_completed      (MetaStartupSequence *seq);
 
 META_EXPORT
-const char * meta_startup_sequence_get_name           (MetaStartupSequence *sequence);
+const char * meta_startup_sequence_get_name           (MetaStartupSequence *seq);
 
 META_EXPORT
-int          meta_startup_sequence_get_workspace      (MetaStartupSequence *sequence);
+int          meta_startup_sequence_get_workspace      (MetaStartupSequence *seq);
 
 META_EXPORT
-uint64_t     meta_startup_sequence_get_timestamp      (MetaStartupSequence *sequence);
+uint64_t     meta_startup_sequence_get_timestamp      (MetaStartupSequence *seq);
 
 META_EXPORT
-const char * meta_startup_sequence_get_icon_name      (MetaStartupSequence *sequence);
+const char * meta_startup_sequence_get_icon_name      (MetaStartupSequence *seq);
 
 META_EXPORT
-const char * meta_startup_sequence_get_application_id (MetaStartupSequence *sequence);
+const char * meta_startup_sequence_get_application_id (MetaStartupSequence *seq);
 
 META_EXPORT
-const char * meta_startup_sequence_get_wmclass        (MetaStartupSequence *sequence);
+const char * meta_startup_sequence_get_wmclass        (MetaStartupSequence *seq);
 
 META_EXPORT
-void        meta_startup_sequence_complete           (MetaStartupSequence *sequence);
+void        meta_startup_sequence_complete           (MetaStartupSequence *seq);
 
 #endif /* META_STARTUP_NOTIFICATION_H */
